prime-router/src/main/kotlin/fhirengine/utils/FHIRReader.kt

```diff
@@ -0,0 +1,45 @@
+package gov.cdc.prime.router.fhirengine.utils
+
+import gov.cdc.prime.router.ActionLogger
+import gov.cdc.prime.router.InvalidReportMessage
+import org.apache.logging.log4j.kotlin.Logging
+import org.hl7.fhir.r4.model.Bundle
+import java.io.BufferedReader
+import java.io.StringReader
+
+/**
+ * Converts raw FHIR data (ndjson Newline Delimited Json) to Bundle objects.
+ */
+class FHIRReader(private val actionLogger: ActionLogger) : Logging {
+    /**
+     * Returns one or more bundles read from the raw FHIR data.
+     */
+    fun getBundles(rawMessage: String): List<Bundle> {
```

Comment:
Can you add the arguments to the kdoc description - Something like 
"Returns one or more bundles read from the FHIR data in [rawMessage]."

Reply:
Yes, needs full kdoc. Missing @ return